modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Sink.scala

```diff
@@ -122,7 +122,7 @@ object Sink {
     producer: KinesisProducerClient[F],
     data: AttributedData[Array[Byte]]
   ): F[Unit] = {
-    val retryPolicy = capDelay[F](config.backoffPolicy.maxBackoff, exponentialBackoff[F](config.backoffPolicy.minBackoff))
+    val retryPolicy = capDelay[F](config.backoffPolicy.maxBackoff, fullJitter[F](config.backoffPolicy.minBackoff))
```

Comment:
Will this be retried forever?

Reply:
yeah, `exponentialBackoff` retries forever as well, the only difference is jitter

Reply:
Right. So only removing `Async[F].raiseError(exception)` below makes sure that the retries are done forever ?

Reply:
I think so but I'm doing another stress test now, I'll update here once I'm done with it, so that we are confident in the change.

Reply:
I can confirm that removing raiseError here allows retries to continue forever

Reply:
Awesome !

Reply:
great finding, then it is a decision of who should control retry policy, our codebase via cats-retry (when recordTtl is low) or KPL (when recordTtl is high)? in both cases we'll have indefinite retry.

Reply:
As I see it it's better to use directly KPL for the retry, as it removes a layer